Source/Lib/ASM_AVX2/EbNoiseExtractAVX2.c

```diff
@@ -8,16 +8,16 @@
 #include "immintrin.h"
 #include "EbUtility.h"
 
-EB_EXTERN EB_ALIGN(16) const uint8_t filter_type[] = {
+static EB_ALIGN(16) const uint8_t filter_type[] = {
     1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4
 };
 
-EB_EXTERN EB_ALIGN(16) const uint8_t weak_chroma_filter[2][32] = {
+static EB_ALIGN(16) const uint8_t weak_chroma_filter[2][32] = {
         { 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4, 2, 4 },
         { 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2 },
 };
 
-inline void eb_vp9_luma_weak_filter_avx2_intrin(
+static void eb_vp9_luma_weak_filter_avx2_intrin(
```

Comment:
`inline` should not be removed. Can we use `static inline`?

Reply:
Sure, I can add inline here